.cicd/scripts/srw_unittest.sh

```diff
@@ -0,0 +1,32 @@
+#!/user/bin/env bash
+#
+# A test script for running the SRW application unittest tests that
+# should be tested on-prem.
+#
+set -e -u -x
+
+script_dir="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" > /dev/null 2>&1 && pwd)"
+
+# Get repository root from Jenkins WORKSPACE variable if set, otherwise,
+# set relative to script directory.
+declare workspace
+if [[ -n "${WORKSPACE}" ]]; then
+  workspace="${WORKSPACE}"
+else
+  workspace="$(cd -- "${script_dir}/../.." && pwd)"
+fi
+
+# Only run this on machines with hpss access
+hpss_machines=( jet hera )
+if [ $hpss_machines =~ ${SRW_PLATFORM} ] ; then
```

Comment:
There appears to be to issues.  The first is the following message:

`./srw_unittest.sh: line 21: [: =~: binary operator expected`

This message no longer appears when an extra [ ] are introduced:

`if [[ $hpss_machines =~ ${SRW_PLATFORM} ]] ; then`

The other issue is that it only checks against Jet, not Jet and Hera:

```
+ hpss_machines=(jet hera)
+ [[ jet =~ hera ]]
[Michael.Lueken@hfe10 scripts]$
```

The following logic should allow both entries to be checked:

```suggestion
for hpss_machine in "${hpss_machines[@]}" ; do
  if [[ $hpss_machine =~ ${SRW_PLATFORM} ]] ; then
```

Reply:
I think perhaps, we'd not want to loop over the list but to just make sure that SRW_PLATFORM is in the list, as long as the list is expanded appropriately.

Reply:
Modifying the lines as:

```suggestion
if [[ ${SRW_PLATFORM} =~ jet|hera ]] ; then
```

should allow the new `srw_unittest.sh` to run on Hera and Jet.